specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/galleryExamples/GalleryImageVersion_Delete.json

```diff
@@ -8,8 +8,23 @@
     "galleryImageVersionName": "1.0.0"
   },
   "responses": {
-    "200": {},
-    "202": {},
-    "204": {}
+    "200": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/000000-8de3-42e7-b00e-8970003527749a/providers/Microsoft.Compute/locations/eastus/capsOperations/0012b61c-2d36-40bc-b7ed-1f0e48757277?api-version=2018-06-01",
```

Comment:
can you use a more recent api version in all of these, such as 2022-03-03 instead of 2018-06-01?

Reply:
updated